velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -31,6 +31,15 @@ class StringTest : public SparkFunctionBaseTest {
     return evaluateOnce<int32_t>("ascii(c0)", arg);
   }
 
+  std::optional<int32_t> bitLength(std::optional<std::string> arg) {
```

Comment:
Would you move these functions as lambdas into corresponding test methods?

Reply:
I've moved the functions related to bit_length. I can do the others in a separate PR?

Reply:
Raised a PR here - https://github.com/facebookincubator/velox/pull/9066. Will rebase once this is merged and publish.